constants/filters.ts

```diff
@@ -41,21 +24,16 @@ const TYPES_FILTERS: LabeledFilters = {
   }
 }
 
-const ORDERED_LOCATION_FILTERS: Array<Filter> = [
-  LOCATION_FILTERS.america,
-  LOCATION_FILTERS.asia,
-  LOCATION_FILTERS.europe,
-  LOCATION_FILTERS.africa
-]
-
 const ORDERED_TYPE_FILTERS: Array<Filter> = [
   TYPES_FILTERS.camp,
   TYPES_FILTERS.unconference,
   TYPES_FILTERS.hackathon
 ]
 
+const LOCATION_CATEGORIES: WorldLocation[] = ['Americas', 'Asia Pacific', 'Europe', 'Africa', 'Online']
```

Comment:
Please, add the notice we talked about offline for following-up your refactor ideas and not to loose track.